packages/release-please/tsconfig.json

```diff
@@ -0,0 +1,15 @@
+{
+  "extends": "gts/tsconfig-google",
+  "compilerOptions": {
+    "esModuleInterop": true,
```

Comment:
It's weird to me that we turned this on for our bots, but don't use it with the rest of our modules.  Can we consider dropping this?

Reply:
Per the offline discussion, this seems to be necessary as one of our dependencies requires it? @bcoe

Reply:
let's check how much trouble it creates removing the setting, and if it's a big refactor, create a tracking ticket to stop using the interop?

I agreed with @JustinBeckwith's reasoning, but might be worth fixing in all bots in a future PR.

Reply:
Filed https://github.com/googleapis/repo-automation-bots/issues/62 to track esModuleInterop